connectors/webhook-connector/element-templates/webhook-connector-intermediate.json

```
@@ -0,0 +1,207 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Webhook Connector",
+  "id": "io.camunda.connectors.webhook.WebhookConnectorIntermediate.v1",
+  "version": 1,
+  "description": "Configure webhook to receive callbacks",
+  "documentationRef": "https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/http-webhook/",
+  "category": {
+    "id": "connectors",
+    "name": "Connectors"
+  },
+  "appliesTo": [
+    "bpmn:IntermediateCatchEvent",
+    "bpmn:IntermediateThrowEvent"
+  ],
+  "elementType": {
+    "value": "bpmn:IntermediateCatchEvent",
+    "eventDefinition": "bpmn:MessageEventDefinition"
+  },
+  "groups": [
+    {
+      "id": "endpoint",
+      "label": "Webhook Configuration"
+    },
+    {
+      "id": "activation",
+      "label": "Activation"
+    },
+    {
+      "id": "variable-mapping",
+      "label": "Variable Mapping"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:webhook:1",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.type"
+      }
+    },
+    {
+      "type": "Hidden",
+      "generatedValue": {
+        "type": "uuid"
+      },
+      "binding": {
+        "type": "bpmn:Message#property",
+        "name": "name"
+      }
+    },
+    {
+      "type": "Hidden",
+      "value": "ConfigurableInboundWebhook",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.subtype"
+      }
+    },
+    {
+      "label": "Webhook ID",
+      "type": "String",
+      "group": "endpoint",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.context"
+      },
+      "description": "The webhook ID is a part of the URL"
+    },
+    {
+      "id": "shouldValidateHmac",
+      "label": "HMAC Authentication",
```

Let's try to keep label names normilized

```suggestion
      "label": "HMAC authentication",
```